src/platform/unittest/TlsTest.cpp

```diff
@@ -64,6 +70,40 @@ struct TlsTest : public ::testing::TestWithParam<bool>
         ClientCertParams = (QUIC_CREDENTIAL_CONFIG*)CxPlatGetSelfSignedCert(CXPLAT_SELF_SIGN_CERT_USER, TRUE);
         ASSERT_NE(nullptr, ClientCertParams);
         ClientCertParams->Flags |= QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION;
+#endif
+#ifndef _WIN32
+        CertParams = (QUIC_CREDENTIAL_CONFIG*)malloc(sizeof(QUIC_CREDENTIAL_CONFIG));
+        if (CertParams) {
```

Comment:
```suggestion
        CertParams = (QUIC_CREDENTIAL_CONFIG*)malloc(sizeof(QUIC_CREDENTIAL_CONFIG));
        ASSERT_NE(nullptr, CertParams);
```
Then you can remove the if block. Same for other allocation failures. Also, please use `CXPLAT_ALLOC_NONPAGED` for allocations here.

Reply:
updated with CXPLAT_ALLOC_NONPAGED and ASSERT_NE.